geode-core/src/test/java/org/apache/geode/management/internal/security/TestCommand.java

```diff
@@ -171,28 +171,37 @@ private static void init() {
     createTestCommand("alter disk-store --name=foo --region=xyz --disk-dirs=bar");
     createTestCommand("destroy disk-store --name=foo", clusterManageDisk);
 
-    // DurableClientCommands
+    // CloseDurableClientCommand, CloseDurableCQsCommand, CountDurableCQEventsCommand,
+    // ListDurableClientCQsCommand
     createTestCommand("close durable-client --durable-client-id=client1", clusterManageQuery);
     createTestCommand("close durable-cq --durable-client-id=client1 --durable-cq-name=cq1",
         clusterManageQuery);
     createTestCommand("show subscription-queue-size --durable-client-id=client1", clusterRead);
     createTestCommand("list durable-cqs --durable-client-id=client1", clusterRead);
 
-    // ExportIMportSharedConfigurationCommands
+    // ExportImportSharedConfigurationCommands
     createTestCommand("export cluster-configuration --zip-file-name=mySharedConfig.zip",
         clusterRead);
     createTestCommand("import cluster-configuration --zip-file-name=value.zip", clusterManage);
 
-    // FunctionCommands
+    // DestroyFunctionCommand, ExecuteFunctionCommand, ListFunctionCommand
+    // TODO PSR: the `destroy function` command is interactive (in its interceptor) when both
```

Comment:
Have got a resolution on these TODO's?

Reply:
Killed the TODO, left the function commented out since it'll throw an error before authentication in that test.

I think that and the associated classes need a refactor in any case.  I'll address the commented-out-ness of the several functions then.